Feature/odd571 access page

deoyani opened the pull request:
Added Access to Data page
Added symbol for access level.
Added accessPage links.

@RayPlante  Check Jira ticket for different records examples.
http://mml.nist.gov:8080/browse/ODD-571

RayPlante commented:
@deoyani @GRG2 
A few tweaks:
- Can you add vertical space between "This data is public" (and its accompanying rights statement) and "Data Available..."?
- Change the wording of "Data Available..." to "Data is available via the following locations:"  (note trailing colon)

deoyani commented:
Made following updates.
Added vertical space between "This data is public" (and its accompanying rights statement) and "Data Available..."
Changed the wording of "Data Available..." to "Data is available via the following locations:" 
Changed "This needs access permissions" to "This data has access restrictions."
Used same sans-serif font for link text as used on the rest of the page.
